bedrock/mozorg/templates/mozorg/mpl/historical.html

```diff
@@ -36,10 +36,10 @@
             <dd>Since legal documents are difficult for non-lawyers to read, Mozilla has in the past provided a version of the license which contains informal explanations of various sections of the license, in the form of hyperlinks.  Note that <em>these explanations are not the license.</em> The explanations are not a legal document, or legal advice. If you need to know <em>exactly</em> what the license requires, you need to read and understand the license itself; if you need legal advice, you need to talk to a lawyer. The annotations are merely a good faith effort to explain the license in simpler language.</dd>
 
             <dt><a href="NPL/1.0/">Netscape Public License, version 1.0</a></dt>
-            <dd>This is the license under which the Mozilla source code was initially released. It is also available in <a href="NPL/1.0/index.txt">plain text</a>. The NPL is obsolete and should not be used.</dd>
+            <dd>This is the license under which the Mozilla source code was initially released. It is also available in <a href="http://website-archive.mozilla.org/www.mozilla.org/mpl/MPL/1.0/index.txt">plain text</a>. The NPL is obsolete and should not be used.</dd>
```

Comment:
This is referencing NPL/1.0/index.txt not MPL/1.0/index.txt.
However since NPL is archived wondering if the url would be similar except point to NPL instead?